diff -NurpP --minimal keepalived-1.1.19.a/configure.in keepalived-1.1.19.b/configure.in
--- keepalived-1.1.19.a/configure.in	2009-09-28 12:56:54.000000000 +0200
+++ keepalived-1.1.19.b/configure.in	2009-10-18 23:00:33.000000000 +0200
@@ -51,7 +51,7 @@ AC_CHECK_LIB(ssl, SSL_CTX_new,,AC_MSG_ER
 AC_CHECK_LIB(popt, poptGetContext,,AC_MSG_ERROR([Popt libraries is required]))
 
 dnl ----[ Kernel version check ]----
-CPPFLAGS="$CPPFLAGS -I$kernelinc"
+dnl CPPFLAGS="$CPPFLAGS -I$kernelinc"
 AC_MSG_CHECKING([for kernel version])
 AC_TRY_RUN([
     #include <stdlib.h>
@@ -116,7 +116,7 @@ AC_SUBST(KERN)
 dnl ----[ Checks for LVS and VRRP support ]----
 IPVS_SUPPORT="_WITHOUT_LVS_"
 if test "$enable_lvs" != "no"; then
-  AC_CHECK_HEADER([net/ip_vs.h],
+  AC_CHECK_HEADER([linux/ip_vs.h],
     [IPVS_SUPPORT="_WITH_LVS_"],
     [
       IPVS_SUPPORT="_WITHOUT_LVS_"
diff -NurpP --minimal keepalived-1.1.19.a/keepalived/include/check_data.h keepalived-1.1.19.b/keepalived/include/check_data.h
--- keepalived-1.1.19.a/keepalived/include/check_data.h	2009-09-28 13:31:07.000000000 +0200
+++ keepalived-1.1.19.b/keepalived/include/check_data.h	2009-10-18 23:04:13.000000000 +0200
@@ -39,7 +39,7 @@
 #else
 #define SCHED_MAX_LENGTH IP_VS_SCHEDNAME_MAXLEN
 #endif
-#include <net/ip_vs.h>
+#include <linux/ip_vs.h>
 #else
 #define SCHED_MAX_LENGTH   1
 #endif
diff -NurpP --minimal keepalived-1.1.19.a/keepalived/include/ipvswrapper.h keepalived-1.1.19.b/keepalived/include/ipvswrapper.h
--- keepalived-1.1.19.a/keepalived/include/ipvswrapper.h	2009-09-28 13:05:16.000000000 +0200
+++ keepalived-1.1.19.b/keepalived/include/ipvswrapper.h	2009-10-18 23:03:04.000000000 +0200
@@ -46,7 +46,7 @@
 #elif _KRNL_2_6_
   #include "../libipvs-2.6/libipvs.h"
 #endif
-  #include <net/ip_vs.h>
+  #include <linux/ip_vs.h>
 #endif
 
 #ifndef IP_VS_TEMPLATE_TIMEOUT
diff -NurpP --minimal keepalived-1.1.19.a/keepalived/libipvs-2.4/libipvs.h keepalived-1.1.19.b/keepalived/libipvs-2.4/libipvs.h
--- keepalived-1.1.19.a/keepalived/libipvs-2.4/libipvs.h	2009-09-28 12:56:54.000000000 +0200
+++ keepalived-1.1.19.b/keepalived/libipvs-2.4/libipvs.h	2009-10-18 23:05:54.000000000 +0200
@@ -10,7 +10,7 @@
 #ifndef _LIBIPVS_H
 #define _LIBIPVS_H
 
-#include <net/ip_vs.h>
+#include <linux/ip_vs.h>
 
 /*
  * The default IPVS_SVC_PERSISTENT_TIMEOUT is a little larger than average
diff -NurpP --minimal keepalived-1.1.19.a/keepalived/libipvs-2.6/libipvs.h keepalived-1.1.19.b/keepalived/libipvs-2.6/libipvs.h
--- keepalived-1.1.19.a/keepalived/libipvs-2.6/libipvs.h	2009-09-28 12:56:54.000000000 +0200
+++ keepalived-1.1.19.b/keepalived/libipvs-2.6/libipvs.h	2009-10-18 23:05:04.000000000 +0200
@@ -10,7 +10,7 @@
 #ifndef _LIBIPVS_H
 #define _LIBIPVS_H
 
-#include <net/ip_vs.h>
+#include <linux/ip_vs.h>
 
 #define MINIMUM_IPVS_VERSION_MAJOR      1
 #define MINIMUM_IPVS_VERSION_MINOR      1
